apex_launchtest/apex_launchtest/util/proc_lookup.py

```diff
@@ -89,14 +89,20 @@ def resolveProcesses(info_obj, *, process=None, cmd_args=None, strict_proc_match
     :param cmd_args: Optional.  If the process param is a string, the cmd_args will be used to
     disambiguate processes with the same name.  cmd_args=None will match all command arguments.
     cmd_args=apex_launchtest.asserts.NO_CMD_ARGS will match a process without command-line
-    arguments
+    arguments.  Otherwise, give arguments as an array of strings
 
     :param strict_proc_matching:  Optional.  If the process param is a string and matches multiple
     processes, strict_proc_matching=True will raise an error
 
     :returns: A list of one or more matching processes taken from the info_obj.  If no processes
     in info_obj match, a NoMatchingProcessException will be raised.
     """
+    # Old versions of this let you pass a string for cmd_args, but there was no way to search
+    # for multiple arguments when you did that.  For backwards compatibility, we'll wrap string
+    # cmd_args in a list
+    if isinstance(cmd_args, str):
```

Comment:
@pbaughman I don't see a test to make sure this works. Does it make sense to add one?

Reply:
It's the very last test.

Reply:
As discussed, the last test in test_resolve_process.py, tests this